cime_config/config_grids.xml

```diff
@@ -4480,6 +4480,11 @@
       <map name="ROF2LND_FMAPNAME">lnd/clm2/mappingdata/maps/ne240np4/map_0.1x0.1_nomask_to_ne240np4_nomask_aave_da_c120706.nc</map>
     </gridmap>
 
+    <gridmap lnd_grid="ne256np4.pg2" rof_grid="r0125">
+      <map name="LND2ROF_FMAPNAME">cpl/gridmaps/ne256pg2/map_ne256pg2_to_r0125_mono.200212.nc</map>
+      <map name="ROF2LND_FMAPNAME">cpl/gridmaps/ne256pg2/map_r0125_to_ne256pg2_mono.200212.nc</map>
```

Comment:
"mono" is no longer a valid key in a map name.

Reply:
@brhillman , can you chime in here? Looks like you added this on the eamxx side with this commit:

```
commit b23a4b48fe0fe967a3f3442a4fb52a0ecd9267d5
Author: Benjamin Hillman <bhillma@sandia.gov>
Date:   Fri Feb 2 17:10:48 2024 -0500

    Add grid maps for ne256 with MOSART r0125

diff --git a/cime_config/config_grids.xml b/cime_config/config_grids.xml
index 614eaf0..f877867 100755
--- a/cime_config/config_grids.xml
+++ b/cime_config/config_grids.xml
@@ -4385,6 +4385,11 @@
       <map name="ROF2LND_FMAPNAME">lnd/clm2/mappingdata/maps/ne240np4/map_0.1x0.1_nomask_to_ne240np4_nomask_aave_da_c120706.nc</map>
     </gridmap>
 
+    <gridmap lnd_grid="ne256np4.pg2" rof_grid="r0125">
+      <map name="LND2ROF_FMAPNAME">cpl/gridmaps/ne256pg2/map_ne256pg2_to_r0125_mono.200212.nc</map>
+      <map name="ROF2LND_FMAPNAME">cpl/gridmaps/ne256pg2/map_r0125_to_ne256pg2_mono.200212.nc</map>
+    </gridmap>
```

Reply:
@rljacob what are we calling the conservative monotonic tempest remap algorithm now?

Reply:
See https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/1217757434/Mapping+file+algorithms+and+naming+convention

Its traave since its a pg2 grid.

Reply:
@brhillman , @rljacob , so should this be changed to:
`map_ne256pg2_to_r0125_traave.200212.nc`
`map_r0125_to_ne256pg2_traave.200212.nc`